share/man/man4/ahci.4

```diff
@@ -149,12 +149,21 @@ statuses in some way (IBPI standard) to report them using present indicators.
 .Sh HARDWARE
 The
 .Nm
-driver supports AHCI compatible controllers having PCI class 1 (mass storage),
-subclass 6 (SATA) and programming interface 1 (AHCI).
+driver supports the following AHCI compatible controllers:
+.Bl -column "Legacy-PATA + AHCI-SATA via ata(4)" "Programming Interface 0x01"
+.It Type: Ta Code:
+.It Mass Storage Ta PCI Class 0x01
+.It SATA Ta Subclass 0x06
+.It AHCI Ta Programming Interface 0x01
+.It Legacy-PATA + AHCI-SATA via ata(4) Ta Programming Interface 0x01
```

Comment:
This is not a list of compatible controllers.  Those are requisites that must be met same time for the driver to attach.

Reply:
Yeah, as I said above I don't think we can really improve this one.